### DISCUSSION TITLE
4.0.1

### DISCUSSION THREAD

**Discussion by nalexandru** [nalexandru/BlenderCompat#71] [Announcements]
Blender 4.0.1, commit [d0dd92834a08](https://projects.blender.org/blender/blender/commit/d0dd92834a08).
Build environment:

    Visual Studio 2019, version 16.11.32
    Windows SDK 10.0.22621
    NVIDIA CUDA 11.8
    NVIDIA OptiX 7.3

AMD HIP and SM3.0 kernel binaries copied from the official release.

The required OpenGL version is now 4.3 and Intel HD4000 support was dropped.
Please read the Graphics Cards section of the release notes: [https://wiki.blender.org/wiki/Reference/Release_Notes/4.0](https://wiki.blender.org/wiki/Reference/Release_Notes/4.0)

<hr /><em>This discussion was created from the release <a href='https://github.com/nalexandru/BlenderCompat/releases/tag/4.0.1'>4.0.1</a>.</em>

**Comment by FilB5** [comment]
![изображение](https://github.com/nalexandru/BlenderCompat/assets/130336541/60d4f797-d684-4db5-bae2-8ae8e94d0b18)

Thank you very much for your work. I really appreciate it as a user win 7.
There is an error in the latest versions, a problem with import - export of FBX, OBJ

**Reply by nalexandru** [reply]
I have reproduced the issue on a clean install of Windows 7 with Service Pack 1.
The problem was resolved after installing the Windows 7 May 2016 rollup, [KB3125574](https://www.catalog.update.microsoft.com/Search.aspx?q=KB3125574). You may need to install the April 2015 Servicing Stack Update before the rollup, [KB3020369](https://www.catalog.update.microsoft.com/Search.aspx?q=KB3020369).
So, the solution is to install Service Pack 1 and the May 2016 rollup (i would recommend installing all updates released for Windows 7).

**Reply by nalexandru** [reply]
I don't think that installing the updates i linked will disable your activation of Windows. You don't need to use Windows Update; you can download the linked files and the SP1 installer and install them offline.

Regarding Linux, i haven't had any issues with the proprietary NVIDIA driver (i'm using it with GTX 1080 and RTX 3090).
